### PR TITLE
Add flow definitions

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by TrySound** [comment]
Why do you guys commit dist in repo?

File: src/index.js.flow [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { Sink, Scheduler, Source, Disposable } from "most";
+import { Stream, } from "most";
```
Comment (briancavalier) [inline src/index.js.flow]: Our code style uses single quotes for imports (except in TS type def files).  Let's make these single quotes.

Also, remove trailing command in the second import.

File: src/index.js.flow [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+declare export default function multicast<A>(s: Stream<A>): Stream<A>;
+
+declare export class MulticastSource<A> {
+  constructor(source: Source<A>): MulticastSource<A>;
+  run (sink: Sink<A>, scheduler: Scheduler): Disposable<A>;
```
Comment (briancavalier) [inline src/index.js.flow]: Extra space here.  I realize this was probably copied from the TS defs, so we can fix it there too.

**Review by briancavalier — CHANGES_REQUESTED** [review]
Couple of very minor things.  Otherwise, looks good.

**Review by briancavalier — APPROVED** [review]

**Comment by briancavalier** [comment]
Thanks @TrySound!  You are a flow-type-adding machine! 😄

**Review by popeda97 — COMMENTED** [review]
#package.json

**Pull request merged**